cms/templates/js/course-outline.underscore

```diff
@@ -85,11 +85,11 @@ if (is_proctored_exam) {
         <% } %>
                 <% if (xblockInfo.isVertical()) { %>
                     <span class="unit-title item-title">
-                        <a href="<%= xblockInfo.get('studio_url') %>"><%= xblockInfo.get('display_name') %></a>
+                        <a href="<%= xblockInfo.get('studio_url') %>"><%- xblockInfo.get('display_name') %></a>
```

Comment:
@agaylard For Underscore templates, there is no equivalent syntax for making the page safe by default.  Instead, it should be understood that using `<%-` should be the rule, and using `<%=` should be the rare exception, so we'd be switching almost all of these, rather than a few particular ones.

Again, this isn't a requirement for this PR, but it certainly would be helpful if you want to wrap up that work.  Of course, as you know, the Mako and underscore templates are related so if you escape here you need to make sure the data is not escaped when sent from Mako to JavaScript.